docker-compose.prod.yml

```diff
@@ -84,12 +87,13 @@ services:
       LETSENCRYPT_EMAIL: ${LETSENCRYPT_EMAIL:-contato@serenata.ai}
       HTTPS_METHOD: ${HTTPS_METHOD:-redirect}
       VIRTUAL_PROTO: http
+    hostname: nginx
 
   logspout:
     image: gliderlabs/logspout:v3.2.4
     restart: always
     env_file:
-      - .env
+      - contrib/.env
```

Comment:
For production we're using `.env.prod` (in the root dir), [as in other containers in this same file](https://github.com/okfn-brasil/serenata-de-amor/blob/6b4cb7c8367a1436d38adc11029e22cfb8318f08/docker-compose.prod.yml#L38-L39).